### DISCUSSION TITLE
Yet another incompact I found between podman-compose and docker-compose

### DISCUSSION THREAD

**Discussion by loynoir** [containers/podman-compose#625] [General]
I try to switch vscode devcontainer from `docker-compose` to `podman-compose`, but failed because of `podman-compose` does not expand relative path. https://github.com/containers/podman-compose/issues/620

Even using the config expanded by `docker-compose`, still got new `Error: no container with name or ID` when `podman start`.

Seems found another incompact :(

**Comment by white-gecko** [comment]
Due to the generic title of your thread "Yet another incompact I found between podman-compose and docker-compose" I want to ask. Does podman-compose try to be compatible to docker-compose with it cli?
